terraform/cli.go

```diff
@@ -40,9 +43,14 @@ func (c CLI) RunWithEnv(stdout io.Writer, workingDirectory string, args []string
 
 	command.Stdout = io.MultiWriter(stdout, c.outputBuffer)
 	command.Stderr = c.errorBuffer
+	command.Stdin = os.Stdin
 
 	err = command.Run()
 	if err != nil {
+		_, isBuffer := c.errorBuffer.(*bytes.Buffer)
+		if !isBuffer {
+			return fmt.Errorf("command execution failed got: %s", err)
+		}
```

Comment:
Is this conditional necessary?

Perhaps its fine if `fmt.Errorf("command execution failed got: %s stderr:\n %s", err, c.errorBuffer)` returns a string where "stderr:\n" contains nothing on the last line?

Alternately the second `%s` could be enclosed in single-quotes so that it is clear that `STDERR` was empty, like this:

```goland
fmt.Errorf("command execution failed got: %s stderr:\n '%s'", err, c.errorBuffer)
```

Reply:
if `c.errorBuffer` is not an array of bytes this means that it is an `io.MultiWriter` and that the content displayed is not human-readable (the content corresponds in fact to what is displayed on stdout in debug mode, so it is already displayed in this case).